src/sentry/event_manager.py

```diff
@@ -685,6 +703,9 @@ def _get_or_create_release_many(jobs, projects):
             date_added=release_date_added[(project_id, version)],
         )
 
+        if _is_commit_sha(release.version):
```

Comment:
Does the SDK sometimes create the version to be the commit sha?
Or does this only work when engineers configure the SDK to use the sha? (in other words, non-default behaviour)

Reply:
The latter. See API-2788